packages/devtools_app/lib/src/inspector/flutter/inspector_tree_flutter.dart

```diff
@@ -177,21 +203,30 @@ class _InspectorTreeState extends State<InspectorTree>
   Future<void> currentAnimateY;
   Rect currentAnimateTarget;
 
+  // fade-in or fade-out animation for Constraints when toggling Show Constraints button
```

Comment:
make this a `///` comment.
Change the variable name to indicate what the variable is used for so the `///` comment isn't really needed.
For example:
`_constraintDisplayController`